geschaeftsordnung.md

```diff
@@ -0,0 +1,111 @@
+# Geschäftsordnung des Fachschaftsrates IT-Systems Engineering
+
+Diese Geschäftsordnung wurde am 04.04.2016 vom Fachschaftsrat beschlossen und tritt in Kraft, sobald die Satzung der Fachschaft IT-Systems Engineering nach einer entsprechenden Änderung die Einführung einer Geschäftsordnung vorsieht.
+
+
+
+##§ 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für den Fachschaftsrat IT-Systems Engineering am Hasso-Plattner-Institut an der Universität Potsdam.
+
+
+##§ 2 Konstituierung
+
+(1) Der Fachschaftsrat tritt spätestens eine Woche nach der Bekanntgabe der Wahlergebnisse auf Einladung des Wahlausschusses zusammen.
+
+(2) Der Fachschaftsrat wählt aus seiner Mitte:
+
+1. einen Vorsitzenden und ggf. dessen Stellvertretenden
+2. einen Finanzreferenten und ggf. dessen Stellvertretenden
+3. einen Beauftragten für die Vernetzung mit den anderen Organen der Studierendenschaft im Sinne der Satzung der Studierendenschaft der Universität Potsdam
+
+(3) Die in (2) festgelegten Ämter können jeweils durch ein konstruktives Misstrauensvotum des Fachschaftsrates oder nach dem Rücktritt einer Person von diesem Amt neu vergeben werden.
+
+
+##§ 3 Aufgabenverteilung
+
+(1) Der Vorsitzende des Fachschaftsrates ist verantwortlich für die Sitzungsleitung gemäß § 5.
+
+(2) Der Finanzreferent verwaltet die Finanzen der Fachschaft gemäß den Beschlüssen des Fachschaftsrates. Er und der ggf. gewählte Stellvertretende sind zu einzeln Verfügungsberechtigten zu erklären. Der Finanzreferent sollte den Finanz-Workshop des Allgemeinen Studierendenausschusses der Universität Potsdam besuchen.
```

Comment:
Siehe ehemaliger `§ 4c Aufgabenverteilung` der Satzung. Diese Regelung gibt es schon seit vielen Jahren und sie hat sich im FSR als sinnvoll herausgestellt. Anderenfalls wären die beiden Finanzer nur gemeinsam verfügungsberechtigt, was zur Folge hat, dass sämtliche Überweisungen immer von beiden Finanzern gemeinsam getätigt werden müssen (höherer Zeitaufwand und nicht von zu Hause möglich). Selbst wenn einer der Finanzer Gelder veruntreut würde dies auffallen, weil der andere Finanzer und das Finanzreferat des AStAs Einblick in die Kontoauszüge haben und alle Abrechnungen kontrolliert werden.

Es wäre natürlich trotzdem möglich, eine Regelung zu treffen, die dem FSR gestattet, selbst über die Art der Verfügungsberechtigung zu entscheiden.